pages/osx/lp.md

```diff
@@ -0,0 +1,15 @@
+# lp
+
+> print files
+
+- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
+
+'lp -d {{device_name}} {{path/to/filename}}
```

Comment:
Ok, I use device name b/c that it what it is labeled under options and supplies.  I figure that someone may be confused if I say printer_name and they see device name.  What do you think?